node/src/components/consensus/traits.rs

```diff
@@ -19,8 +19,9 @@ impl<VID> ValidatorIdT for VID where VID: Eq + Ord + Clone + Debug + Hash {}
 pub(crate) trait ConsensusValueT:
     Eq + Clone + Debug + Hash + Serialize + DeserializeOwned
 {
+    /// Returns whether the consensus value is empty.
+    fn is_empty(&self) -> bool;
```

Comment:
I'd prefer to call it something like `needs_validation`, so the trait doesn't care about whether these are lists or something else. And it would make it more natural to e.g. set it to always `false` for some tests, or in case requirements in production change.

Reply:
Good idea.

Reply:
Done in [`06fe993` (#653)](https://github.com/CasperLabs/casper-node/pull/653/commits/06fe99365ed796bbdf1d1d3ce0528cac422999cd)